### DISCUSSION TITLE
Standalone Container Names

### DISCUSSION THREAD

**Discussion by gravitarium** [jesseduffield/lazydocker#524] [Q&A]
Hey everyone, I just recently started using lazydocker and absolutely love it. One thing that I noticed and wanted to ask you guys, in the Standalone Container section I'm getting a lot of names for the container taken out of the Label block instead of the image name resulting in a lot of ubi8s or ubi9s:
![image](https://github.com/jesseduffield/lazydocker/assets/167464910/aeae454b-d298-4b82-bb65-f1299cb739c8)

Is there a way to modify that to show the actual name? Thanks in advance for your time and hard work! 

Respectfully,
gravitarium

**Comment by huyz** [comment]
Same here. How can we give `container_name` priority?

**Reply by huyz** [reply]
Relevant: https://github.com/jesseduffield/lazydocker/issues/40
